pyproject.toml

```diff
@@ -28,6 +28,7 @@ stringcase = "^1.2.0"
 docker = "^7.1.0"
 sqlalchemy = "~=1.0"
 requests = "^2.32.0"
+jsonalias = "^0.1.2"
```

Comment:
I'm not sure this is even worth including since its usage is so limited. Can you just include the type in `JsonArtifact`?